Reaqtive/Core/Reaqtive.Linq/Reaqtive/Operators/SingleAsync.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT License.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+namespace Reaqtive.Linq.Reaqtive.Operators
```

Comment:
Abbreviate the namespace to be consistent with all the other operators. I believe VS is trying to be a bit too smart in inferring namespaces from folders here.